code/modules/mob/living/carbon/emote.dm

```diff
@@ -115,12 +115,12 @@
 
 /datum/emote/living/carbon/circle/run_emote(mob/user, params, type_override, intentional)
 	. = ..()
+	if(!length(user.get_empty_held_indexes()))

```

Comment:
Is there really no helper for this ?

Reply:
correct, theres no helper

Reply:
there's helper for if there's a free left or right hand but not for both afaik

Reply:
make one imo